### DISCUSSION TITLE
Weird Output for scores when using fetch_traces()

### DISCUSSION THREAD

**Discussion by kevinpkaria** [langfuse/langfuse#2616] [Support]
Upon using the fetch_trace() function and thne printing the scores, I receive an array of IDs instead of the score values. Why so?
An array of proper scores and not unhelpful IDs would be helpful.
<img width="1512" alt="Screenshot 2024-07-11 at 6 06 15 PM" src="https://github.com/langfuse/langfuse/assets/89715638/24137f27-aef6-4e72-b4d1-f6ffc0f7ab2b">
<img width="1512" alt="Screenshot 2024-07-11 at 6 07 41 PM" src="https://github.com/langfuse/langfuse/assets/89715638/034bfec3-3066-478d-b832-f25b523d9065">

**Comment by marcklingen** [comment]
Agree, the scores included on this endpoint are for backwards compatibility and we want to improve the interface

**Reply by kevinpkaria** [reply]
Is there any way I could derive the scores in the required fromat from this format?

**Comment by bluesky314** [comment]
Hey am facing this same issue. Want to rank the traces by the lowest score and analyse the results on my local environment. @marcklingen Is there any way to do this as its pretty important to dig deeper into failure cases ?

**Reply by marcklingen** [reply]
we plan to add filters to fetch_observations/traces based on scores. Currently not available yet.

**Reply by marcklingen** [reply]
you could query the scores though to make this workflow work: https://api.reference.langfuse.com/#get-/api/public/scores
